tox.ini

```diff
@@ -65,7 +65,8 @@ commands =
     yarn
     make dist
     # Test whl installation
-    pip install {toxinidir}/dist/kolibri_explore_plugin-0.0.1-py2.py3-none-any.whl
+
+    pip install {toxinidir}/dist/kolibri_explore_plugin-0.0.3-py2.py3-none-any.whl
```

Comment:
We should stop doing hotfixes and fix this once and for all, by removing the hardcoded value. Maybe it's just a matter of:

```
    pip install {toxinidir}/dist/kolibri_explore_plugin-*.whl
```

We are tracking this issue in a separate ticket: https://phabricator.endlessm.com/T31414